answering.py

```diff
@@ -0,0 +1,76 @@
+import re
+import json
+from container.answer_loading import load_question_data
+
+
+def parseAnswerTextSpacing(s):
+    # Lowercase version
+    words = re.sub("-", " ", s.lower().strip())
+    words = re.sub("\n+", "\n", words)
+    words = re.sub(" +", " ", words)
+    words = " ".join([w.strip() for w in words.split(" ") if w.strip()])
+    words = words.strip()
+    return words
+
+
+def checkAnsweredQuestions(pageQuestion, answeredQuestions):
+        for answeredQuestion in answeredQuestions:
+            if answeredQuestion.find(pageQuestion) == 0:
+                # and/or other specifications
+                return answeredQuestion
+        return None
+
+
+#### placeholder for data retrieval/recieving endpoint
+def load_question_data_plc(id):
+    import numpy as np
+    import pandas as pd
+    questionData = {}
+    df = pd.read_csv(f"data/"+id+".txt")
+    mtx = np.array(df)
+    for i in range(len(mtx)):
+        row = [ij for ij in list(mtx[i,:]) if str(ij).lower() != "nan"]
+        questionData[row[0]] = []
+        for j in range(1, len(row)):
+            questionData[row[0]] += [row[j]]
+    return questionData
+
+
+def answer_input_questions(id, inputQuestions):
+    
+    ###########
```

Comment:
Not sure what this means, generally remove these kinds of comments

Reply:
I use the #### comments to represent overrides. Updated the comments.